packages/api/src/auth/auth.model.ts

```diff
@@ -0,0 +1,16 @@
+export interface GithubDTO {
+  id: string; // numeric
+  nodeId: string;
+  displayName: string;
+  username: string;
+  profileUrl: string;
+  emails: { value: string }[];
```

Comment:
Why emails and photos are objects with property value inside whereas username / nodeId etc not?

Reply:
Well, that's the way passport.js maps response data. http://www.passportjs.org/docs/profile/

Reply:
Is it not strange, for you :) ? 

Ok, so here you have your auth domain model which is dependent on an external library. 
You should keep your domain model pure, because for example changing of library etc. should not change your logic etc. There is a place for simple mapping, IMO.